operating/manage-user-access/manage-role-based-access-control-3630.adoc

```diff
@@ -0,0 +1,62 @@
+[id="manage-role-based-access-control"]
+= Managing role-based access control in {product-title} 3.63.0 and newer
+include::modules/common-attributes.adoc[]
+:context: manage-role-based-access-control
+
+toc::[]
+
+[role="_abstract"]
+{product-title} (RHACS) comes with role-based access control (RBAC) that you can use to configure roles and grant various levels of access to {product-title} to different users.
```

Comment:
Similar to previous comment:
Maybe "to different users" --> "for different users" so you aren't repeating "to" again? It was sort of hard to read this the first time.